### ISSUE TITLE
Strange Overlay on Right Eye Specular Highlights 0.8.0

### ISSUE THREAD

**Issue by Spikerama** [Nibre/MotherVR#134]
Hi mate. Firstly thank you so much for providing the community with such an amazing mod. Out of all the VR games I play I still maintain that Alien Isolation is the most enjoyable by far and keep coming back to it. That's thanks to you and your tireless effort. Really love everything you have done with this. Thanks a thousand times over. 

Unfortunately I had to remove 0.8.0 and put 0.6.0 back in because of a very strange issue which I couldn't resolve myself. It's kind of hard to explain but I'll try.

When I first started with 0.8.0 in I had a few false starts because of having to reconfigure how steam loaded the VR. There were clear popup warnings obviously developed by you on how to do this. WHen the game ran, my vision was cross eyed. cameras not aligned. I restarted the game and that seemed to fix it but I notice there was a strange, sort of overlay image that appeared in the right eye which only showed when moving over reflective surface. So the rest of the image was clear but it was almost as though I could see through the specular part of the texture. 

I looked at my monitor and noticed a very thin band of image down the right side. I went into VR settings and played around with the mirror mode and that seemed to fix it on the monitor. Think I changed it to Left Eye. Alas, it didn't fix it in the HMD and not mater what config I tried it was still  the same. Anyway thought I'd share that with you. Don't know if anyone else has replicated it but felt you should know.

**Comment by ElricOfMelnibone** [comment]
Getting the same thing here in the HMD (Vive), Left Eye vs Right Eye reflections/specular/whatever. I tried turning various graphics options on and off, but nothing helped.
Easy place to see it, if you go to the Sevastopol Galleria, up on the 2nd floor and look down at the middle set of stairs where you see reflective metal banding, in one eye you'll see consecutive metal bands with no gaps in between, and in the other eye you'll see one band, then a hug gap, then the next band. You'll also notice it around the window sills in that area, etc. I mean, the effect is a lot of places, but that's where it really started to stick out to me.

**Comment by ElricOfMelnibone** [comment]
Someone else posted in the 0.8 [Reddit thread](https://www.reddit.com/r/Vive/comments/9y0f3a/mothervr_beta_080_alien_isolation_vr_mod_vr/) and provided a screenshot:
https://imgur.com/a/mEt24dw

**Comment by Spikerama** [comment]
Yes that's exactly where I noticed it first. In the transport terminals. 

I wonder what's causing it if only a few people have replicated the issue.

**Comment by ElricOfMelnibone** [comment]
Definitely the right eye that's off. I finally noticed when looking at some decals on overhead bins or something in one section that the numbers were absent in the right eye and present in the left.
And oh god, the reactor (alien nest) level, weird overlays and reflections everywhere, I'm guessing because the water reflections everywhere?

**Comment by Spikerama** [comment]
Well if it's happening on the water it may have something to do with Planar Reflections. I will try have a go tonight to see if I can get a similar result and also have a look in the Graphics settings to see if there's a Planar option.

**Comment by ElricOfMelnibone** [comment]
Might be two separate issues there, I dunno. The water thing seemed to be in both eyes IIRC.

**Comment by Spikerama** [comment]
Could be right. I turned on Planar Reflections in settings and it didn't make any difference. Still right eye issue.

**Comment by SimplexPL** [comment]
Did anyone find a solution to this problem? I have it constantly on reflections but also on smoke, lights, even some textures (like grafitti). When I move my head objects pop in and out of view, it's very distracting.

**Comment by Spikerama** [comment]
> Did anyone find a solution to this problem? I have it constantly on reflections but also on smoke, lights, even some textures (like grafitti). When I move my head objects pop in and out of view, it's very distracting.

No I never did. It's still like that and now if I revert back to the earlier MortherVR version it is like that too. What gives???

**Comment by SimplexPL** [comment]
This mod has been abandoned for almost 2 years, perhaps there were some changes to VR runtimes that caused this issue and it needs to be fixed on the mod's side.

**Comment by Bluscream** [comment]
The developer is still working on other VR mods, and will return to AIVR once he finished the other one. He will be able to greatly improve this mod because of a shared codebase

**Comment by ElricOfMelnibone** [comment]
I just re-installed AI, and am not noticing the right-eye issue anymore. It could be the new version, but it could also be one of several other things--I did adjust the gamma up a fair bit, but I also installed some mods ("lens flare remover" and "4K settings"). My current save is in the Galleria, and I am not seeing any of the issues there I mentioned up above. I'll check some more today, but left vs right eye seems fine in that regard. Item/texture pop-ins are still happening, but that's another issue.
I also have in-game supersampling ("pixel aspect") set to 1.0, and Steam's at 2.0, and anti-aliasing off.
FWIW

**Comment by Dan88Man** [comment]
@ElricOfMelnibone Was the right eye issue indeed gone for good, or did it reoccur after further playing? And if it didn't occur anymore, could you please point me to the exact additional mods you mentioned you have installed? Thanks!
Also: For me the issue appears only on higher super sampling settings. Either in Steam or in game with the pixel aspect setting. Are you sure you aren't just on a lower supersampling setting where this issue doesnt occur? For me, the higher the supersampling is, the more pronounced the right eye issue becomes. Below a certain setting, it doesn't occur at all for me, but on that setting the image is too blurry.

**Comment by ElricOfMelnibone** [comment]
@Dan88Man I didn't have any issues for the rest of my playthrough. Here's the lens flare remover: https://www.nexusmods.com/alienisolation/mods/1   Also had this one: https://www.nexusmods.com/alienisolation/mods/6   And this one: https://www.nexusmods.com/alienisolation/mods/12     And here's the page with the 4K settings (note that I turned planar reflections off because they don't look right in VR): https://www.pcinvasion.com/make-alien-isolation-look-amazing-4k8k-shadows-reflections-detail-and-more/     The game looked really good to me at the supersampling rate I was using. I guess it's possible the right-eye issue could be a thing at higher levels, but this was the same rate I had used previously (I never changed it from where I had set it previously in Steam).

**Comment by Dan88Man** [comment]
@ElricOfMelnibone Thanks a lot. Unfortunately installing the mods didn't solve it for me. I left out the mods for more inventory items and for removing a save sound, as I believe they don't do anything to the graphics.

Would you mind sending me those 2 relevant files of yours?

[install dir]\Alien Isolation\DATA\ENGINE_SETTINGS.XML

\Steam\userdata\[user_id]\214490\local\settings.bin

if you don't have the Steam version of AI, the setttings.bin file might be located somewhere else of course.

I don't know if there is any upload option on this website. If not, I hope you could put it on any cloud hoster and post the link here. I would appreciate it very much.

Cheers

**Comment by ElricOfMelnibone** [comment]
@Dan88Man My apologies--I just checked and none of those files remain on my system (I uninstalled AI after playing it earlier this year). Sorry about that...

**Comment by Dan88Man** [comment]
Ok I understand. What graphics card did you use if I may ask? Maybe that's also an important factor.

**Comment by Dan88Man** [comment]
@ElricOfMelnibone Ok I understand. What graphics card did you use if I may ask? Maybe that's also an important factor.

**Comment by ElricOfMelnibone** [comment]
@Dan88Man I'm on a 1080Ti (11GB VRAM).  Funny thing is, the second time I played the game, which was the first time in VR, I was on the same card. Maybe my upping the gamma on the 2nd VR playthrough had some effect here?

**Comment by Dan88Man** [comment]
@ElricOfMelnibone I just tested upping the gamma completely. Didn't solve it.

Were you still on the original Vive on that playthrough? And are you today still on the Vive? From the headset info and the SteamVR super sampling setting you mentioned, I might figure out your render resolution.

But then of course it wouldn't explain that it did occur before on the alleged same settings....

**Comment by ElricOfMelnibone** [comment]
@Dan88Man Good question about the Vive. Both playthroughs were on an OG Vive, but the second one was a replacement after dropping the first one, and I don't know if this factors into things, but the 2nd one has that "dark" lens glitch where one side is blacker than the other. I would have thought that would make things worse, if anything, though it obviously didn't. But maybe there's something in those 4K settings I used?

**Comment by Dan88Man** [comment]
@ElricOfMelnibone I applied all the 4k changes. And also tried Planar on and off. What I noticed though is, when using exactly those code snippets from the site...

https://www.pcinvasion.com/make-alien-isolation-look-amazing-4k8k-shadows-reflections-detail-and-more/

...the game wouldn't even start for me, because it turned out that the website uses a different kind of quotation marks. It even uses different quotation marks for opening and closing. The settings would only be recognized and the game started once I replaced those two types of quotation marks from the site...

”  ″
with this from the original file:

"

Not sure if you also had to go through this, or if the website even had these changes displayed back then.

Do you have the Steam version of AI ? Or Epic version, or any other?

**Comment by ElricOfMelnibone** [comment]
@Dan88Man Never had that issue with the code that I recall, but I may have first pasted it into Notepad or something else that replaced the quotation marks for me.
I was using the Steam version of AI, both times for VR. I also have the boxed version which I played in pancake mode when I first got it.

**Comment by kjr81** [comment]
I've been struggling with this today, the only way I found to get rid of it is to turn pixel scale down to 0.7 but obviously it looks worse, the more you increase the pixel scale from there the more it comes into view. Not sure how to fix, increasing steamvr sampling seems to have no effect on the game.

**Comment by Ratasan64** [comment]
Noticing the exact same here. It's so frustrating. My fav game in vr...
